homeassistant/components/matter/fan.py

```diff
@@ -128,41 +119,29 @@ async def async_set_preset_mode(self, preset_mode: str) -> None:
         if self._attr_preset_mode in [PRESET_NATURAL_WIND, PRESET_SLEEP_WIND]:
             await self._set_wind_mode(None)
 
-        await self.matter_client.write_attribute(
-            node_id=self._endpoint.node.node_id,
-            attribute_path=create_attribute_path_from_attribute(
-                self._endpoint.endpoint_id,
-                clusters.FanControl.Attributes.FanMode,
-            ),
+        await self.write_attribute(
             value=FAN_MODE_MAP[preset_mode],
+            matter_attribute=clusters.FanControl.Attributes.FanMode,
         )
 
     async def async_oscillate(self, oscillating: bool) -> None:
         """Oscillate the fan."""
-        await self.matter_client.write_attribute(
-            node_id=self._endpoint.node.node_id,
-            attribute_path=create_attribute_path_from_attribute(
-                self._endpoint.endpoint_id,
-                clusters.FanControl.Attributes.RockSetting,
-            ),
+        await self.write_attribute(
             value=self.get_matter_attribute_value(
                 clusters.FanControl.Attributes.RockSupport
             )
             if oscillating
             else 0,
```

Comment:
Let's wrap multiline if statements in parenthesis

Reply:
addressed